components/input/style/mixin.less

```diff
@@ -23,8 +23,7 @@
     border-color: @hoverBorderColor;
   }
   & when not (@theme = variable) {
-    box-shadow: @input-outline-offset @outline-blur-size @outline-width
-      fade(@borderColor, @outline-fade);
+    box-shadow: @input-outline-offset @outline-blur-size @outline-width fade(@borderColor, @outline-fade);
```

Comment:
Please move styles to `patch.less`, we don't change any less file which is synced from `ant-design`

Reply:
done, sorry for the late fix